pkg/proxy/apis/config/fuzzer/fuzzer.go

```diff
@@ -33,7 +33,7 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 	return []interface{}{
 		func(obj *kubeproxyconfig.KubeProxyConfiguration, c fuzz.Continue) {
 			c.FuzzNoCustom(obj)
-			obj.BindAddress = fmt.Sprintf("%d.%d.%d.%d", c.Intn(256), c.Intn(256), c.Intn(256), c.Intn(256))
+			obj.NodeIPOverride = []string{fmt.Sprintf("%d.%d.%d.%d", c.Intn(256), c.Intn(256), c.Intn(256), c.Intn(256))}
```

Comment:
should possibly fuzz to 0-2 values...
(@aojea did we end up adding an ip-address-fuzzing util somewhere? I feel like we talked about that as part of the ServiceCIDR stuff)

Reply:
BindAddress is not meant to be comma-separated, If I fuzz between 0-2 then I will lose data in roundtrip conversion tests.

Reply:
Yeah... I think we want to just give up on the idea that you can roundtrip v1alpha1 with a fuzzing step in the middle. We'll want that to work for v1alpha2, but the goal here is to explicitly freeze-and-abandon the v1alpha1 API.

(I'm not sure what this means for the unit tests...)